src/logplayer/player.h

```diff
@@ -51,8 +51,10 @@ class Player : public QThread
     Network* m_referee;
     Network* m_vision;
     Network* m_legacyVision;
+    Network* m_index;
```

Comment:
index is not sent via network. Have you mixed things up? Shouldn't there be a Network instance for visionTracker? 🤔